pkg/backup/interface.go

```diff
@@ -4,21 +4,25 @@ import (
 	"context"
 
 	"github.com/kubevela/workflow/api/v1alpha1"
+	"github.com/kubevela/workflow/pkg/backup/sls"
 )
 
-// PersistType is the type of persister.
-type PersistType string
-
 const (
 	// PersistTypeSLS is the SLS persister.
-	PersistTypeSLS PersistType = "sls"
+	PersistTypeSLS string = "sls"
 )
 
 // NewPersister is a factory method for creating a persister.
-func NewPersister(persistType PersistType) persistWorkflowRecord {
+func NewPersister(persistType string, config map[string][]byte) persistWorkflowRecord {
 	switch persistType {
 	case PersistTypeSLS:
-		return &slsHandler{}
+		return &sls.Handler{
+			LogStoreName:    string(config["LogStoreName"]),
```

Comment:
We should encode it first since the value is from secret.

Reply:
Nit: it's better if we can make config variable name like `LogStoreName` to const.